enterprise/core-edge/src/main/java/org/neo4j/commandline/dbms/UnbindFromClusterCommand.java

```diff
@@ -0,0 +1,184 @@
+import org.neo4j.commandline.admin.IncorrectUsage;
+import org.neo4j.commandline.admin.OutsideWorld;
+import org.neo4j.helpers.Args;
+import org.neo4j.io.fs.DefaultFileSystemAbstraction;
+import org.neo4j.io.fs.FileSystemAbstraction;
+import org.neo4j.io.fs.FileUtils;
+import org.neo4j.kernel.StoreLockException;
+import org.neo4j.kernel.internal.StoreLocker;
+
+import static java.lang.String.format;
+import static java.nio.file.Files.exists;
+import static org.neo4j.coreedge.core.EnterpriseCoreEditionModule.CLUSTER_STATE_DIRECTORY_NAME;
+
+public class UnbindFromClusterCommand implements AdminCommand
+{
+    public static class Provider extends AdminCommand.Provider
+    {
+        public Provider()
+        {
+            super( "unbind" );
+        }
+
+        @Override
+        public Optional<String> arguments()
+        {
+            return Optional.of( "[--database=<name>] " );
+        }
+
+        @Override
+        public String description()
+        {
+            return "Removes cluster state data from the specified database making it suitable for use non-Core-Edge " +
+                    "databases";
+        }
+
+        @Override
+        public AdminCommand create( Path homeDir, Path configDir, OutsideWorld outsideWorld )
+        {
+            return new UnbindFromClusterCommand( homeDir, new DefaultFileSystemAbstraction() );
+        }
+    }
+
+    private Path homeDir;
+    private FileSystemAbstraction fsa;
+
+    UnbindFromClusterCommand( Path homeDir, FileSystemAbstraction fsa )
+    {
+        this.homeDir = homeDir;
+        this.fsa = fsa;
+    }
+
+    @Override
+    public void execute( String[] args ) throws IncorrectUsage, CommandFailed
+    {
+        Args parsedArgs = Args.parse( args );
+
+        try
+        {
+            Path pathToSpecificDatabase =
+                    homeDir.resolve( "data" ).resolve( "databases" ).resolve( databaseNameFrom( parsedArgs ) );
+            confirmTargetDatabaseDirectoryExists( pathToSpecificDatabase );
+            confirmClusterStateDirectoryExists( Paths.get( pathToSpecificDatabase.toString(), "cluster-state" ) );
+            confirmTargetDirectoryIsWritable( pathToSpecificDatabase );
+            deleteClusterStateIn( clusterStateFrom( pathToSpecificDatabase ) );
+        }
+        catch ( IllegalArgumentException e )
+        {
+            throw new IncorrectUsage( e.getMessage() );
+        }
+        catch ( UnbindFailureException e )
+        {
+            throw new CommandFailed( "Unbind failed: " + e.getMessage(), e );
+        }
+    }
+
+    private Path clusterStateFrom( Path target )
+    {
+        return Paths.get( target.toString(), CLUSTER_STATE_DIRECTORY_NAME );
+    }
+
+    private void confirmTargetDirectoryIsWritable( Path dbDir ) throws CommandFailed
+    {
+        Path lockFile = dbDir.resolve( StoreLocker.STORE_LOCK_FILENAME );
+        if ( exists( lockFile ) )
+        {
+            if ( Files.isWritable( lockFile ) )
```

Comment:
You fail silently if the lockfile is unwritable (i.e. if the current user doesn't have the correct permissions to operate on the database). Presumably that's not what you want for this command. This case will happen, for example, with the Debian package if the operator forgets to `sudo -u neo4j`.